packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -150,6 +150,9 @@ export interface AttachmentDownloadResult {
     blobUrl: string;
 }
 
+// @beta
+export type AttachmentMetadata = FileMetadata | /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */ InlineImageMetadata;
```

Comment:
Optional: Fixing the camel casing `MetaData` everywhere

Reply:
`Metadata` is the correct form. It is one word and not two.